src/pat/tooltip/tooltip.js

```diff
@@ -26,7 +26,7 @@ define([
     parser.addArgument("height", "auto", ["auto", "max"]);
     parser.addArgument("trigger", "click", ["click", "hover"]);
     parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
-    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
+    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
```

Comment:
@pilz good catch. i must have changed that like like 10 times before pushing.
@cornae currently if text is passed as we will insert that text into <p> element ... now with content-html we insert this in <div/> element. 

i do agree content option should be also handling html as well as text, but you know where you've been using it and how much can go wrong.

safe option until then would be to add content-html option